packages/framework/src/Model/AdminNavigation/BreadcrumbOverrider.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Model\AdminNavigation;
+
+class BreadcrumbOverrider
+{
+    /**
+     * @var string|null
+     */
+    protected $lastItemLabel;
+
+    /**
+     * @param string $lastItemLabel
+     */
+    public function overrideLastItem(string $lastItemLabel): void
```

Comment:
I did consider it, but in the end the fact that this name is shorter and that this could be extended by more optional parameters without breaking BC won. I think it's still understandable very easily.